tasks/tasks.go

```diff
@@ -0,0 +1,194 @@
+package tasks
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"os/exec"
+	"strings"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/joyent/containerpilot/utils"
+)
+
+// TaskConfig configures tasks that run periodically
+type TaskConfig struct {
+	Name      string   `json:"name"`
+	Args      []string `json:"command"`
+	Frequency string   `json:"frequency"`
+	Timeout   string   `json:"timeout,omitempty"`
+
+	freqDuration    time.Duration
+	timeoutDuration time.Duration
+	cmd             *exec.Cmd
+	quitChannel     chan int
```

Comment:
I think some of it can be adapted to use the Pollable interface - but the PollTime is an integer - not a duration - so it assumes Seconds.

Can we alter the API to supply a Duration instead?